lib/chat/chat_messenger.dart

```diff
@@ -199,9 +237,9 @@ class ChatMessengerState extends State<ChatMessenger> {
 }
 
 class ChatMessenger extends StatefulWidget {
+  final String peerId;
   final String recipient;
-  final List<Object> messages;
-  ChatMessenger({Key key, @required this.recipient, this.messages = dummyData}) : super(key: key);
+  ChatMessenger({Key key, @required this.peerId, @required this.recipient}) : super(key: key);
```

Comment:
Is `peerId` required? Im getting `missing_required_param` errors in the following files:

<img width="551" alt="Screen Shot 2020-07-10 at 12 15 49 PM" src="https://user-images.githubusercontent.com/25041263/87185577-3ef82c00-c2a7-11ea-940d-f6805b363a51.png">